backend/src/test/java/com/github/hamideh6182/controller/BookControllerTest.java

```diff
@@ -186,4 +186,37 @@ void increaseBookQuantityTest_WhenQuantityPlusOne() throws Exception {
                                                                                                 }
                                                 """));
     }
+
+    @Test
+    @DirtiesContext
+    void decreaseBookQuantityTest_WhenQuantityMinusOne() throws Exception {
```

Comment:
Maybe you could add a "then" part to the test name, where you declare what output you expect.